documentation/bolt.ditamap

```diff
@@ -15,6 +15,7 @@
         <topicref href="bolt_versioning.md" format="markdown"/>
         <topicref href="upgrading_to_bolt_3.md" format="markdown"/>   
         <topicref href="bolt_known_issues.md" format="markdown"/>
+        <topicref href="glossary.md" format="markdown"/>
```

Comment:
This should go under `Reference`, not here.

Reply:
I told Arthur to put it here... mostly cos of precedence (the Puppet docs). But the reference section is fine 👍

Reply:
It could also just go at the root level, right at the bottom (or just above reference)...
If we want it to be a bit more visible

Reply:
@hestonhoffman If we decide it's best in root, could you make a suggestion for that change? (not sure where to put it lol)

Reply:
Yeah sure thing!

Reply:
@artlawson I can't suggest a change cos it's out of the PR scope, but I think you can just put it right at the bottom, after Troubleshooting :)